tests/upgrades/test_performance_tuning.py

```diff
@@ -122,14 +179,54 @@ def _create_custom_hiera_file(mongodb_type, tune_size="default"):
             if tune_size == "medium":
                 fd.write("\n".join(MEDIUM_TUNING_DATA))
 
+    @staticmethod
+    def _data_collection_of_set_tune_params(
```

Comment:
If this def does `create` and `read` then does it worth naming it as collection ?